vim_onenote.ahk

```diff
@@ -196,41 +295,70 @@ Send, {ShiftUp}
 return
 
 ; Delete current line
+; dd handled specially, to delete newline.
 d::
-if IsLastKey("d")
-{
-    Send, {Home}{ShiftDown}{End}
-    Send, ^c ; Yank before delete, don't use cut so blank lines are deleted 
+    SelectMotion()
+    send ^x
+    if IsLastKey("d")
     Send, {Del}
-    Send, {Shift}
-}
-return 
+return
 
-+s::
++S::
 Send, {Home}{ShiftDown}{End}
 Send, ^x ; Cut instead of yank and delete
 Send, {ShiftUp}
 Gosub, InsertMode   
 return 
 
-return 
 ; TODO handle regular paste , vs paste something picked up with yy
```

Comment:
Ahh, that must have been what I meant.  Good reason to take care writing your comments for the next time you come to read them.